File: abc/405/cpp/a.cpp
#include <iostream>

using namespace std;

int sv1(int r, int x){
        if (1600 <= r && r <= 2999 && x == 1){
        return 1;
    }
    if (1200 <= r && r <= 2399 && x == 2){
        return 1;
    }
    return 0;
}

int main(void){
    int r, x, n;
    cin >> r >> x;
    n = sv1(r, x);
    if (n == 1){
        cout << "Yes";
    } else {
        cout << "No";
    }
  
    return 0;
}

File: abc/405/py/a.py
def sv1(r: int, x: int) -> str:
    if 1600 <= r <= 2999 and x == 1:
        return 'Yes'
    if 1200 <= r <= 2399 and x == 2:
        return 'Yes'
    return 'No'

def main():
    r, x = map(int, input().split())
    res = sv1(r, x)
    print(res)

if __name__ == "__main__":
    main()
